docs/developer-docs/latest/getting-started/introduction.md

```diff
@@ -6,38 +6,38 @@ canonicalUrl: https://docs.strapi.io/developer-docs/latest/getting-started/intro
+- [`strapi/strapi-docker`](https://github.com/strapi/strapi-docker): the code used to generate the official Docker images for Strapi (available through our [Docker Hub](https://hub.docker.com/r/strapi/strapi)).
+- [`strapi/awesome-strapi`](https://github.com/strapi/awesome-strapi): all plugins managed by the core team as well as contributions built by the community. It is used as a central place to find and submit new packages such as plugins, middlewares, hooks, and general enhancements to the core of Strapi.
 
 ## Strapi Community
 
-Strapi is a community-oriented project with an emphasis on transparency. The Strapi team has at heart to share their vision and build the future of Strapi with the Strapi community. This is why the [roadmap](https://portal.productboard.com/strapi) is open: as all insights are very important and will help steer the project in the right direction, any community member is most welcome to share ideas and opinions there.
+Strapi is a community-oriented project with an emphasis on transparency. The Strapi team has at heart to share their vision and build the future of Strapi with the Strapi community. This is why the [roadmap](https://portal.productboard.com/strapi) is open: as all insights are very important and help steer the project in the right direction, any community member is most welcome to share ideas and opinions there.
 
-Community members also take great part in providing the whole community a plethora of resources about Strapi. You can find [tutorials](https://strapi.io/tutorials/) on the Strapi website, where you can also create your own. Also, as an open-source project, the technical documentation of Strapi is open to contributions (see [Open-source & Contribution](#open-source-contribution)).
+Community members also take great part in providing the a plethora of resources about Strapi. You can find [tutorials](https://strapi.io/tutorials/) on the Strapi website, where you can even create your own. Also, as an open-source project, the technical documentation of Strapi is open to contributions (see [Open-source & Contribution](#open-source-contribution)).
 
 ::: strapi Want to join the community?
-You can join [GitHub](https://github.com/strapi/strapi), the [Forum](https://forum.strapi.io/), and the [Discord](https://discord.strapi.io) to share your ideas and opinions with other community members and members of the Strapi team. If you're looking for news and updates about Strapi, [Twitter](https://twitter.com/strapijs) and the [blog](https://strapi.io/blog) are pretty good places to start!
+You can join [GitHub](https://github.com/strapi/strapi), the [Forum](https://forum.strapi.io/), and the [Discord](https://discord.strapi.io) to share your ideas and opinions with other community members and members of the Strapi team. If you are looking for news and updates about Strapi, [Twitter](https://twitter.com/strapijs) and the [blog](https://strapi.io/blog) are pretty good places to start!
```

Comment:
this is just a standard contraction, either work and there is no correct form on this. `you're` and `you are` is the exact same thing.

Reply:
This is an example representative for a bigger problem, for which you yourself provide an example in the previous comment: "[...] the community org will take it's place" should be "[...] the community org will take its place" ("its" without apostrophe).
Other examples are situations where words should just be plural forms instead of "form's" or where there is no real possessive form fiven (like "tennis shoes", "sport cars", "Archilles heel" ...)
There are teams out there who decided to therefore not use contractions. It forces writers to pay more attention to this detail.
But of course that also would mean that examples like this one would also have to be rewritten, even though they are totally fine. 
Not sure, how strict the Strapi guidelines are, but I wanted to point out the potential problem.

Reply:
For sure if there are bugs in the usage of contractions then they need to be fixed but that shouldn't mean removing all contractions all together, that just seems heavy handed.

Usage of contractions is a natural way of writing and isn't generally "forbidden" in technical writing that I'm aware of (given I'm not a technical writer, but am a native english speaker. It's my one and only language :sweat_smile: )

Reply:
We do try to avoid contractions but if there are some in the documentation it’s still fine, it’s not an absolute technical writing rule honestly and it's quite complicated to fix all contractions everywhere in the Strapi documentations.
Regarding the other examples you mentioned, those are probably typos - feel free to fix them through other contributions, they're always most welcome 😄